recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "solvation_analysis" %}
-{% set version = "0.1.5" %}
+{% set version = "0.3.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/solvation_analysis-{{ version }}.tar.gz
```

Comment:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/solvation-analysis-{{ version }}.tar.gz
```
try this

Reply:
I think you can also do `{{ name.replace("-", "_" }}`, but it's been a while since I've played with variables that way.